cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go

```diff
@@ -49,7 +49,7 @@ func runControlPlane() func(c workflow.RunData) error {
 			return errors.New("control-plane phase invoked with an invalid data struct")
 		}
 
-		// if this is not a control-plande node, this phase should not be executed
+		// if this is not a control-planed node, this phase should not be executed
```

Comment:
Please change to `control-plane`

Reply:
@yagonobre thanks, i have modified it.
would you like to add ok-to-test and lgtm label?